src/XenCert/StorageHandler.py

```diff
@@ -265,11 +269,7 @@ def mp_config_verification_tests(self):
             #2. Create and plug SR
             printout("CREATING SR")
             (retval, sr_ref, device_config) = self.create()
-            if not retval:                    
-                raise Exception("      SR creation failed. ")
-            else:
-                display_operation_status(True)
```

Comment:
why display_operation_status is removed?

Reply:
added.